### PR TITLE
Feature/sst 70: Save Position

### PR DESCRIPTION
**Pull request by S1lver-Owl**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
Save global position

## Fixes
Link all the related issues from the issue tracker
- Closes #

## How has this been tested ?
(If possible) describe the steps to test these modifications.

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: behaviors/sonia_base_behaviors/src/sonia_base_behaviors/save_pose_sm.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	'''
+	Save Pose
+	'''
+
+
+	def __init__(self):
+		super(SavePoseSM, self).__init__()
+		self.name = 'Save Pose'
+
+		# parameters of this behavior
```
Comment (CaroBoud) [inline behaviors/sonia_base_behaviors/src/sonia_base_behaviors/save_pose_sm.py:33]: Do we want to keep those comments ?

File: behaviors/sonia_stack_behaviors/manifest/test_save_pose.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	<executable package_path="sonia_stack_behaviors.test_save_pose_sm" class="TestSavePoseSM" />
+	<tagstring></tagstring>
+	<author>Nimai</author>
+	<date>Sat Jun 24 2023</date>
+	<description>
+		Test Save Pose
+	</description>
+
+
+	<!-- Contained Behaviors -->
```
Comment (CaroBoud) [inline behaviors/sonia_stack_behaviors/manifest/test_save_pose.xml:14]: Do we want to keep the comments ?

File: behaviors/sonia_stack_behaviors/src/sonia_stack_behaviors/test_save_pose_sm.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+	'''
+	Test Save Pose
+	'''
+
+
+	def __init__(self):
+		super(TestSavePoseSM, self).__init__()
+		self.name = 'Test Save Pose'
+
+		# parameters of this behavior
```
Comment (CaroBoud) [inline behaviors/sonia_stack_behaviors/src/sonia_stack_behaviors/test_save_pose_sm.py:36]: Commentaires nécessaires ?

File: behaviors/sonia_base_behaviors/src/sonia_base_behaviors/save_pose_sm.py (hunk 1 above)
Comment (S1lver-Owl) [reply behaviors/sonia_base_behaviors/src/sonia_base_behaviors/save_pose_sm.py:33]: auto generated

File: behaviors/sonia_stack_behaviors/manifest/test_save_pose.xml (hunk 2 above)
Comment (S1lver-Owl) [reply behaviors/sonia_stack_behaviors/manifest/test_save_pose.xml:14]: auto generated

File: behaviors/sonia_stack_behaviors/src/sonia_stack_behaviors/test_save_pose_sm.py (hunk 3 above)
Comment (S1lver-Owl) [reply behaviors/sonia_stack_behaviors/src/sonia_stack_behaviors/test_save_pose_sm.py:36]: auto generated

**Review by CaroBoud — APPROVED** [review]
Seems good to me !

**Pull request merged**